### ISSUE TITLE
Binding events to validate

### ISSUE THREAD

**Issue by Afunakwa** [chronotruck/vue-ctk-date-time-picker#15]
So I'm not quite sure if this is an issue or not, but I'd like to fire an event when the user clicks validate, and I can't see any obvious way to configure the dateTime picker in order to do so

**Comment by LouisMazel** [comment]
Hi, 

Why you want an event for this ? The validate button is just here for close the composant. 

If you want fire the change of value you can use the `input` event like this : 

```
<ctk-date-time-picker 
  @input="yourMethod($event)"
/>

*$event = new value (= v-model)

**Comment by Devvvvv** [comment]
> Hi,
> 
> Why you want an event for this ? The validate button is just here for close the composant.
> 
> If you want fire the change of value you can use the `input` event like this :
> 
> ```
> <ctk-date-time-picker 
>   @input="yourMethod($event)"
> />
> ```
> *$event = new value (= v-model)

But i want use this picker for loading information from server. Make a request with every data update is wrong! When the user has selected the desired date and clicked "apply" you need to make a request. For this i need a handler for "apply" button

**Comment by LouisMazel** [comment]
Hi,

Ok I understand ! 

I released a beta version with this feature (2.0.0-rc3)
You can test it : 
`npm install vue-ctk-date-time-picker@beta`
DEMO : https://htmlpreview.github.io/?https://github.com/chronotruck/vue-ctk-date-time-picker/blob/feature/refont/demo/index.html
DOC : https://github.com/chronotruck/vue-ctk-date-time-picker/blob/feature/refont/README.md

Le me know if you have others feedbacks (events, bugs, etc)

Thanks
